lib/ngtcp2_unreachable.c

```diff
@@ -35,6 +35,7 @@
 #endif /* WIN32 */
 
 void ngtcp2_unreachable_fail(const char *file, int line, const char *func) {
+#ifdef HAVE_UNISTD_H
```

Comment:
This breaks windows build.  See line62.

Reply:
Thanks for pointing it out, I've added a common define to sort it out.

Reply:
Thank you. Looks good.